Marlin/printcounter.h

```diff
@@ -132,6 +132,12 @@ class PrintCounter: public Stopwatch {
     void showStats();
 
     /**
+     * @brief Return the currently loaded statistics
+     * @details Return the raw data, in the same structure used internally
+     */
+    inline printStatistics getStats() { return this->data; }
```

Comment:
You don't have to use `inline` here, as you're defining the body of the method inside the header the compiler with automatically inline this for you.